### PR TITLE
Add a 3.10.4 vanilla save to test cases

### PR DESCRIPTION
**Pull request by chennin**
Add an updated vanilla save from 3.10.4. It was generated on a huge galaxy by letting observer mode run for 300 years (then saved in non-observer mode).  This save specifically tests both accented characters and names literally `=`, ie, #126. It also has newer features like council positions.

Also modifies the test case to remove all *.db files output by the test.

⚠️ Please merge **after** #124 and #128

### CODE REVIEW

File: test/parser_test.py [hunk 1]
```diff
@@ -30,8 +30,7 @@ def test_real_save():
...
     from pathlib import Path
 
-    output_db = Path(
-        f"{config.CONFIG.base_output_path}/db/nexitronawareness_1329922464.db"
-    )
     cli.f_parse_saves(save_path="test/saves")
-    output_db.unlink(missing_ok=True)
+    output_db_dir = Path(f"{config.CONFIG.base_output_path}/db")
+    for db in list(output_db_dir.glob('*.db')):
+      db.unlink(missing_ok=True)
```
Comment (eliasdoehne) [inline test/parser_test.py:36]: I think this risks deleting db files from real games when running tests which I think we should avoid. 

Instead, we can use the tmp_path fixture [provided by pytest](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) and let the OS handle the cleanup. Then, I would also parametrize the test to a test case per game ID so we can easier figure out what broke

It should work like this:

```
@pytest.mark.parametrize(
    "save_dir",
    [path for path in pathlib.Path(__file__).parent.glob(r"saves/*")],
    ids=lambda path: path.stem,
)
def test_real_save(save_dir, tmp_path):
    from stellarisdashboard import cli, config

    debug = config.CONFIG.debug_mode
    base_path = config.CONFIG.base_output_path

    config.CONFIG.debug_mode = True
    config.CONFIG.base_output_path = tmp_path
    try:
        cli.f_parse_saves(save_path=save_dir)
    finally:
        config.CONFIG.debug_mode = debug
        config.CONFIG.base_output_path = base_path
```

Looks like this (before merging https://github.com/eliasdoehne/stellaris-dashboard/pull/128):
image

**Comment by eliasdoehne** [comment]
I like the idea of testing the code with a real save file, but I think we'll need a different way to store the files since they will quickly increase the size of the repo if we add new ones on every release. 

Maybe we can just move them to a separate repo and clone it during the github action workflow. Or something like this? https://docs.github.com/en/repositories/working-with-files/managing-large-files 
(I never tried it)

@chennin @MichaelMakesGames Do you have any ideas for this?

File: test/parser_test.py (hunk 1 above)
Comment (eliasdoehne) [reply test/parser_test.py:36]: (maybe it also makes sense to organize the save files by game version, e.g. `test/saves/3.10.4/kaanvisamfriendship3_-449354374/2508.07.01.sav`)

Comment (chennin) [reply test/parser_test.py]: Good point, thank you. Updated

**Comment by chennin** [comment]
> @chennin @MichaelMakesGames Do you have any ideas for this?

@eliasdoehne I tried Git LFS and it seems it won't work for the usual model of collaborators using forks to code and open PRs.

```
batch response: @chennin can not upload new objects to public fork chennin/stellaris-dashboard
error: failed to push some refs to 'github.com:chennin/stellaris-dashboard.git'
```

**Comment by chennin** [comment]
Closing this in favor of an upcoming different solution

**Pull request closed without merging**